### PR TITLE
[13.0][FIX] stock_barcodes: Do not distribute lots for non tracking products for inventories

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

ping @chienandalu

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
:+1:

File: stock_barcodes/wizard/stock_barcodes_read_inventory_views.xml [hunk 1]
```diff
@@ -29,12 +29,6 @@
                                             <field name="location_ids" />
                                         </div>
                                     </div>
-                                    <div class="row">
```
Comment (pedrobaeza) [inline stock_barcodes/wizard/stock_barcodes_read_inventory_views.xml:32]: Why removing this?

**Review by pedrobaeza — APPROVED** [review]

File: stock_barcodes/wizard/stock_barcodes_read_inventory_views.xml (hunk 1 above)
Comment (chienandalu) [reply stock_barcodes/wizard/stock_barcodes_read_inventory_views.xml:32]: Not very practical. When there are many references it takes to much space in the screen

Comment (pedrobaeza) [reply stock_barcodes/wizard/stock_barcodes_read_inventory_views.xml:32]: Ok, but then do a separate commit with proper commit message explaining that or at least comment it in the same commit

**Pull request merged**